packages/rum-core/test/specHelper.ts

```diff
@@ -277,46 +271,6 @@ export const noopRecorderApi: RecorderApi = {
   getReplayStats: () => undefined,
 }
 
-// Duration to create a cookie lasting at least until the end of the test
-const COOKIE_DURATION = ONE_MINUTE
-
-export function mockSyntheticsWorkerValues(
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** nice job lifting this, I feel this file is a little long